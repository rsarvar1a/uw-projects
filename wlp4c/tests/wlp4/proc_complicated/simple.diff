2d1
< 361
