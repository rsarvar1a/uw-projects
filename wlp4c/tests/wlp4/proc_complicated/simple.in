8
10 20 40 60 90 120 160 200

