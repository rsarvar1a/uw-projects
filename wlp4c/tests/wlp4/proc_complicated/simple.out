Enter length of array: Enter array element 0: Enter array element 1: Enter array element 2: Enter array element 3: Enter array element 4: Enter array element 5: Enter array element 6: Enter array element 7: Running MIPS program.
361
0
MIPS program completed normally.
$01 = 0x00000ed8   $02 = 0x00000008   $03 = 0xfffffff6   $04 = 0x00000004   
$05 = 0x00000010   $06 = 0xffffffeb   $07 = 0x00000000   $08 = 0x00000008   
$09 = 0x00000ed8   $10 = 0x00000000   $11 = 0x00000001   $12 = 0x00000000   
$13 = 0x00000000   $14 = 0x00000000   $15 = 0x00000000   $16 = 0x00000000   
$17 = 0x00000000   $18 = 0x00000000   $19 = 0x00000000   $20 = 0x00000000   
$21 = 0x00000000   $22 = 0x00000000   $23 = 0x00000000   $24 = 0x00000000   
$25 = 0x00000000   $26 = 0x0000094c   $27 = 0x00000b24   $28 = 0x000007b4   
$29 = 0x01000000   $30 = 0x01000000   $31 = 0x8123456c   
